gcp_variant_transforms/transforms/limit_write_test.py

```diff
@@ -0,0 +1,101 @@
+# Copyright 2018 Google Inc.  All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Tests for limit_write module."""
+
+from __future__ import absolute_import
+
+import unittest
+
+from apache_beam.testing.test_pipeline import TestPipeline
+from apache_beam.testing.util import assert_that
+from apache_beam.testing.util import equal_to
+from apache_beam.transforms import Create
+
+from gcp_variant_transforms.beam_io import vcfio
+from gcp_variant_transforms.transforms import limit_write
+from gcp_variant_transforms.transforms.variant_to_bigquery import _WRITE_SHARDS_LIMIT as DefaultShardLimit
```

Comment:
It's a bit odd to reference this here. Perhaps just hard code 4500 in your test? I don't think using the exact value matters much here.